code/__DEFINES/dcs/signals/signals_client.dm

```diff
@@ -0,0 +1,4 @@
+/// from base of [/client/proc/handle_spam_prevention] (message, mute_type)
+#define COMSIG_CLIENT_AUTOMUTE_CHECK "client_automute_check"
```

Comment:
Rename to `MOB_`, and put it in the mob signals file.

Reply:
Done.